src/components/pickers/RangeSlider/styled.ts

```diff
@@ -0,0 +1,114 @@
+import { tasty } from '../../../tasty';
+
+export const StyledThumb = tasty({
+  styles: {
+    top: '2x',
+    width: '2x',
+    height: '2x',
+    fill: {
+      '': '#purple',
+      hovered: '#6B4FEE',
+      dragged: '#7A77FF',
```

Comment:
Colors look like tokens that we have.

Reply:
<img width="406" alt="image" src="https://user-images.githubusercontent.com/105391768/192372799-3c417868-2926-4b33-955d-2daba2828b4f.png">

```js
const colors = {
  pink: '255, 100, 146',
  'pink-02': '255, 131, 168',
  purple: '122, 119, 255',
  'purple-text': '102, 93, 232',
  'purple-01': '122, 119, 255',
  'purple-02': '149, 146, 255',
  'purple-03': '175, 173, 255',
  'purple-04': '202, 201, 255',
  text: '91, 92, 125',
  dark: '20, 20, 70',
  'dark-01': '20, 20, 70',
  'dark-02': '67, 67, 107',
  'dark-03': '114, 114, 144',
  'dark-04': '161, 161, 181',
  'dark-05': '213, 213, 226',
  'grey-light': '248, 248, 249',
  light: '246, 246, 248',
  white: '255, 255, 255',
  black: '0, 0, 0',
  'danger-text': '227, 0, 42',
  danger: '255, 55, 73',
  'light-grey': '248, 248, 249',
  success: '49, 195, 124',
  'success-text': '48, 166, 102',
  note: '251, 188, 5',
};
```
I didn't found matches for `hovered` state inside tokens. :(

Reply:
@nikolaykost Check Figma again. Sergey fixed colors.